tests/integration/create_message_batches/test_field_validation.py

```diff
@@ -10,7 +10,7 @@
 }
 INVALID_MESSAGE_VALUES = ["", [], 5, 0.1]
 INVALID_NHS_NUMBER = ["999054860", "99905486090", "abcdefghij", "", [], {}, 5, 0.1]
-INVALID_DOB = ["1990-10-1", "1990-1-10", "90-10-10", "10-12-1990", "1-MAY-2000", "1990/01/01", "", [], {}, 5, 0.1]
+INVALID_DOB = ["1990-10-1", "1990-1-10", "90-10-10", "10-12-1990", "1-MAY-2000", "1990/01/01", "", [], {}, 5, 0.1, None]
```

Comment:
can we add these into the documentation for the test cases that test for this? add them into a table much like the correlation id partial does